packages/x-date-pickers-pro/src/internal/hooks/useStaticRangePicker/useStaticRangePicker.types.ts

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {
+  ExportedPickersViewLayoutSlotsComponent,
+  ExportedPickersViewLayoutSlotsComponentsProps,
+  DateOrTimeView,
+  BaseNextPickerProps,
+  UsePickerParams,
+  PickersSlotsComponent,
+  PickersSlotsComponentsProps,
+  ExportedBaseToolbarProps,
+} from '@mui/x-date-pickers/internals';
+import { DateRange } from '../../models/range';
+
+export interface UseStaticRangePickerSlotsComponent
+  extends ExportedPickersViewLayoutSlotsComponent,
+    Pick<PickersSlotsComponent, 'PaperContent'> {}
+
+export interface UseStaticRangePickerSlotsComponentsProps
+  extends ExportedPickersViewLayoutSlotsComponentsProps,
+    Pick<PickersSlotsComponentsProps, 'paperContent'> {
+  toolbar?: ExportedBaseToolbarProps;
+}
+
+export interface StaticRangeOnlyPickerProps {
```

Comment:
We already have this type defined in `StaticOnlyPickerProps`.
Maybe it would make sense to `Pick` it from there instead of re-declaring?

Reply:
Indeed, and we have small inconsistencies between desktop and range desktop pickers as well, extending would prevent those

Reply:
Done